tensorboard/plugins/scalar/scalars_plugin.py

```diff
@@ -119,6 +121,25 @@ def scalars_impl(self, ctx, tag, run, experiment, output_format):
         else:
             return (values, "application/json")
 
+    def _scalars_multirun_impl(self, ctx, tag, runs, experiment):
+        """Result of the form `(body, mime_type)`."""
+        all_scalars = self._data_provider.read_scalars(
+            ctx,
+            experiment_id=experiment,
+            plugin_name=metadata.PLUGIN_NAME,
+            downsample=self._downsample_to,
+            run_tag_filter=provider.RunTagFilter(runs=runs, tags=[tag]),
+        )
+        result = {}
+        for run in all_scalars:
+            scalars = all_scalars.get(run, {}).get(tag, [])
```

Comment:
nit: The fallback in `all_scalars.get(run, {})` is superfluous, since
we iterate `for run in all_scalars`. The fallback in `….get(tag, [])`
is, I think, also superfluous, since `run` won’t appear in the output at
all if it doesn’t have data for `tag` (since your `RunTagFilter` has a
singleton `tag` axis). So this could just be `all_scalars[run][tag]`,
which I think is clearer because the current version suggests codepaths
that cannot actually exist.